cmd/x-check-mailserver/config.go

```diff
@@ -14,6 +17,13 @@ func newNodeConfig(fleet string, networkID uint64) (*params.NodeConfig, error) {
 		return nil, err
 	}
 
+	cleanPrivKey := strings.TrimPrefix(privKey, "0x")
+	if len(cleanPrivKey) == 64 {
+		c.NodeKey = cleanPrivKey
+	} else if len(cleanPrivKey) != 64 && len(cleanPrivKey) != 0 {
+		return nil, errors.New("Wrong private key length! Expected 64 char hexadecimal.")
```

Comment:
Not a big thing, but just so you know, there is a golang convention for writing error messages. They don't begin with uppercase letters and they don't end in any punctuation.